src/insns/malformed_clear_tag.adoc

```diff
@@ -0,0 +1,2 @@
+NOTE: This instruction sets `cd.tag=0` if `cs1` 's bounds are <<section_cap_malformed,malformed>>, as malformed bounds
+cause both the base and top bounds to be zero which is not legal in a tagged capability.
```

Comment:
The base and top bounds can be zero in a tagged capability.

Reply:
fixed